spec/models/post_spec.rb

```diff
@@ -48,4 +48,4 @@
       expect(@post1.five_recent_comments.count).to eql(0)
```

Comment:
- [ ] Some of the model specs are failing. Let's ensure all model specs are passing.

**Failing specs:**
![image](https://github.com/Abe1able/Bloggingly/assets/12745474/68da7eef-6c65-4247-a191-ae05ffac9125)

Reply:
Fixed, thank you very much.